lib/src/constants.dart

```diff
@@ -151,6 +176,14 @@ class _Constant implements ConstantReader {
   const _Constant(this._object);
 
   @override
+  dynamic get anyValue =>
+      _object.toBoolValue() ??
+      _object.toIntValue() ??
+      _object.toStringValue() ??
+      _object.toListValue() ??
+      _object.toMapValue();
+
+  @override
```

Comment:
You're missing `symbol` here – it's supported on `DartObject`

You'll just want to wrap it in `new Symbol` if it returns a String.